### ISSUE TITLE
NTR: PhyH activity low in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7938]
New term request PhyH activity low in fibroblasts (segal_060322164047): Phytanoyl-CoA hydroxylase

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000131
- label: Reduced phytanic acid oxidase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:2433405
- def: Acitivity of phytanic acid oxidase below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7938
- synonyms: PhyH activity low in fibroblasts(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal
I see this was a duplicate with https://github.com/obophenotype/human-phenotype-ontology/issues/8182
Please disregard this entry. The correct  PhyH term is HP:4000163
